docs/pages/foundations/typography.vue

```diff
@@ -133,15 +134,15 @@ For macOS, we use Apple system font San Francisco, you can download from [Fonts
 
 ## Formatting
 
-| Style                                                                                                                     | Notes                                                      |
-|:--------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------|
-| *Italics text*                                                                                                            | Using `<em>`                                               |
-| **This is bold text**                                                                                                     | Using `<strong>`                                           |
-| <ol><li>This is an ordered list;</li><li>Second item of ordered list;</li><li>Third item of ordered list.</li></ol>       | `<li>` element has margin `8px` with default `line-height` |
-| <ul><li>This is an unordered list;</li><li>Second item of unordered list;</li><li>Third item of unordered list.</li></ul> | same as ordered list                                       |
+| Style                                                                                                                     | Notes                                                    |
```

Comment:
This was because smart ass ESlint consider `<li>` as an opened tag so I had to change it to `li`.